packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -34,26 +34,26 @@ $status-indicator-size: 1em * 0.5;
     height: $status-indicator-size;
     margin-inline-end: $spacing-xs;
```

Comment:
@jeffbelser-okta I could use some help here. Our spacing and font-size values are generated currently. Whether it's via a function or Sass math, we don't have a template for converting these to Tokens.

Reply:
@edburyenegren-okta We can use style dictionary to generate values in a similar fashion, [this example](https://github.com/amzn/style-dictionary/tree/main/examples/advanced/transitive-transforms) shows transforms for color but the same idea would hold true for type scales or spacing scales. We have access to a node run time after all and can do some scripting.

This all said, I would suggest this as a quicker fix: just define the resulting values (after all the math) as new tokens and use those. When we need to make changes in the future we'll have to update all the values to make sure the scale remains intact. Not great but the result is the same. Eventually we can build transitive transformed tokens to get our scale logic back in place

Reply:
Nice! I should have assumed that'd be possible.

[This issue](https://github.com/amzn/style-dictionary/issues/365) gives some options that don't require custom transforms. I'll play around - if nothing generative works as expected in our current setup, I'll drop in the hard values for now.